authors/apps/profiles/urls.py

```diff
@@ -12,5 +15,8 @@
     path("follow/following", ProfileFollowingAPIView.as_view(),
          name="following"),
     path("follow/followers", ProfileFollowersAPIView.as_view(),
-         name="followers")
+         name="followers"),
+    path("", UserProfileListAPIView.as_view(), name="profile-list"),
+    path("<username>/", ProfileRetrieveUpdateView.as_view(),
```

Comment:
@pbkabali great work please get rid of the trailing slash

Reply:
@oma0256, I don't know how this came about but let me work on it. Thanks

Reply:
worked on.